### PR TITLE
Feat: XCM config & service

### PR DESCRIPTION
**Pull request by tuul-wq**
- XCM service + tests
- Load XCM config on app start and transfer page (+ browser cache to reduce network activity)
- Save XCM config

by @Asmadek  & @tuul-wq

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ const config: Config = {
     '^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|json)$)': '<rootDir>/scripts/fileTransform.js',
   },
   transformIgnorePatterns: [],
+  testRegex: ['^.*\\.(test|spec)\\.[jt]sx?$'],
```
Comment (tuul-wq) [inline jest.config.ts:41]: This is needed to skip folders like `mock` inside __tests__

File: src/renderer/app/index.tsx [hunk 2]
```diff
@@ -26,11 +27,14 @@ if (!container) {
   throw new Error('Root container is missing in index.html');
 }
 
+kernelModel.events.appStarted();
```
Comment (tuul-wq) [inline src/renderer/app/index.tsx:30]: The fastest way to signal our models that app has started

**Comment by tuul-wq** [comment]
Task linked: [CU-862katqmx Cross-chain transfer config reading and parsing](https://app.clickup.com/t/862katqmx)

**Comment by tuul-wq** [comment]
Task linked: [CU-862kbacfp Cross-chain service](https://app.clickup.com/t/862kbacfp)

**Review by egor0798 — APPROVED** [review]

**Pull request merged**